readme.md

```diff
@@ -15,18 +15,61 @@ Control the Neopixels directly attached to the board.
 ```js
 var Playground = require("playground-io");
 var five = require("johnny-five");
-var board = new five.Board({
-  io: new Playground({
-    port: "/dev/tty.usbmodem1411"
-  })
+var io = new Playground({
+  port: "/dev/tty.usbmodem1411"
 });
+var board = new five.Board({ io: io });
 board.on("ready", function() {
   var pixels = Array.from({ length: 10 }, function(_, index) {
     return new five.Led.RGB({
       controller: Playground.Pixel,
       pin: index
     });
-  });
+  }); 
+  
+  var led = new five.Led(13); 
+  
+  var buttonL = new five.Button('4', {
+    isPullup: true,
+    invert: true
+  }); 
+  
+  var buttonR = new five.Button('19', {
+    isPullup: true,
+    invert: true
+  }); 
+  
+  var toggle = new five.Switch('21'); 
+  
+  var piezo = new five.Piezo({
+    pin: '5',
+    controller: Playground.Piezo
+  }); 
+  
+  var thermometer = new five.Thermometer({
+    controller: "TINKERKIT",
+    pin: "A0",
+    freq: 100
+  }); 
+  
+  var light = new five.Sensor({
+    pin: "A5",
+    freq: 100
+  }); 
+  
+  var sound = new five.Sensor({
+    pin: "A4",
+    freq: 100
+  }); 
+  
+  var accelerometer = new five.Accelerometer({
+    controller: Playground.Accelerometer
+  }); 
+  
+  var tap = new Playground.Tap(io); 
+  
+  var touch = new Playground.CapTouch(io);
```

Comment:
- The `Playground.Accelerometer` controller should emit "tap" events. 
- The `Playground.CapTouch` controller should be `Playground.Touch` and should be written to work with `Touch` class (which is an alias for `Keypad`).